cmd/swarmctl/main.go

```diff
@@ -6,6 +6,7 @@ import (
 	"github.com/docker/swarmkit/cmd/swarmctl/cluster"
 	"github.com/docker/swarmkit/cmd/swarmctl/network"
 	"github.com/docker/swarmkit/cmd/swarmctl/node"
+	"github.com/docker/swarmkit/cmd/swarmctl/secrets"
```

Comment:
nit: This should be named `secret` to be consistent with the rest (e.g. `node`, `network`, ...)